documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -132,17 +175,60 @@ Regardless of which endpoint is being invoked, the URL format for scaling is the
 
 When the operator receives a scaling request, it will:
 
-*	Perform an authentication and authorization check to verify that the specified user is allowed to perform the specified operation on the specified resource.
-*	Validate that the specified domain, identified by `domainUID`, exists.
-*	Validate that the WebLogic cluster, identified by `clusterName`, exists.
-*	Verify that the specified WebLogic cluster has a sufficient number of configured servers or sufficient dynamic cluster size to satisfy the scaling request.
-*	Initiate scaling by setting the `replicas` field within the corresponding Domain, which can be done in either:
-      *	A `cluster` entry, if defined within its cluster list.
-      *	At the domain level, if not defined in a `cluster` entry.
+* Perform an authentication and authorization check to verify that the specified user is allowed to perform the specified operation on the specified resource.
+* Validate that the specified domain, identified by `domainUID`, exists.
+* Validate that the WebLogic cluster, identified by `clusterName`, exists.
+* Verify that the specified WebLogic cluster has a sufficient number of configured servers or sufficient dynamic cluster size to satisfy the scaling request.
+* Verifies that the specified cluster has a corresponding Cluster resource defined or else creates one if necessary.
+* Initiate scaling by setting the `replicas` field within the corresponding Cluster resource.
+
+In response to a change in the `replicas` field in the Cluster resource, the operator will increase or decrease the number of Managed Server instance Pods to match the desired replica count.
 
-In response to a change to either `replicas` field, in the Domain, the operator will increase or decrease the number of Managed Server instance Pods to match the desired replica count.
+#### Supported Autoscaling Controllers
+While continuing to support automatic scaling of WebLogic clusters with the WebLogic Diagnostic Framework (WLDF) and Prometheus, Operator 4.0 now supports the [Kubernetes Horizontal Pod Autoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+##### Kubernetes Horizontal Pod Autoscalar (HPA)
+Automatic scaling of an individual WebLogic cluster, by the Kubernetes Horizontal Pod Autoscalar, is now supported since the Cluster custom resource has enabled the `/scale` subresource. In order to scale based on "resource metrics" (like CPU, memory, or any custom metrics), an implementation of the `Metrics API` must be deployed to the Kubernetes cluster.  One such implementation of the `Metrics API` is the [Kubernetes Metrics Server](https://kubernetes-sigs.github.io/metrics-server/).  Prometheus offers the [Prometheus Adapter](https://github.com/kubernetes-sigs/prometheus-adapter), which implements the Kubernetes resource metrics, custom metrics, and external metrics APIs and is also suitable for use with the Horizontal Pod Autoscaler.
```

Comment:
has enabled the \`/scale\` subresource 

-->

has enabled the Kubernetes \[\`/scale\` subresource\]\(https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource\)